examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -31,10 +31,19 @@ class Task extends TypedEventEmitter<ITaskEvents> implements ITask {
     public set priority(newValue: number) {
         this._priority.set(newValue);
     }
+    public diffName: string;
+    public diffPriority: number;
+    public diffType: string;
+    public DEFAULT_PRIORITY: number = Number.POSITIVE_INFINITY;
+    public DEFAULT_NAME: string = "";
+    public DEFAULT_DIFF_TYPE: string = "none";
     public constructor(
         private readonly _id: string,
         private readonly _name: SharedString,
-        private readonly _priority: ISharedCell<number>
+        private readonly _priority: ISharedCell<number>,
+        diffName: string = "",
+        diffPriority: number = Number.POSITIVE_INFINITY,
+        diffType: string = "none",
```

Comment:
Can we declare these as `public diffaname: string = ""`, etc, to avoid the declaration in lines 34-36 and the initialization in 55-57?